docs/api/en/materials/LineBasicMaterial.html

```diff
@@ -41,7 +41,7 @@ <h2>Examples</h2>
 			[example:webgl_lines_dashed WebGL / lines / dashed]<br />
 			[example:webgl_lines_sphere WebGL / lines / sphere]<br />
 			[example:webgl_materials WebGL / materials]<br />
-			[example:webgl_physics_rope WebGL / phyics / rope]
+			[example:physics_ammo_rope physics / ammo / rope]
```

Comment:
Can you please also fix the Chinese version of the doc page?

Reply:
> Can you please also fix the Chinese version of the doc page?

I have done.